### PR TITLE
Added Jérôme Giacomini

### PR DESCRIPTION
**Pull request by jgiacomini**
:pencil2: **Blog Url**: https://jeromegiacomini.net/Blog/

:scroll: **Feed Url**:  https://jeromegiacomini.net/Blog/feed/

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [x] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by jfversluis** [comment]
I'm a bit worried on the continuation of your blog. Strictly seen you have checked the 'My blog is active with at least 3 Xamarin related blog posts in the last 6 months' but you don't comply with it. Although it seems to exist for a while you don't seem to be very consistent in your activity. @Cheesebaron ?

File: src/Firehose.Web/Authors/JeromeGiacomini.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("https://jeromegiacomini.net/Blog/feed/"); }
+        }
+
+        public GeoPosition Position => new GeoPosition(48.8704842, 2.3449646);
+
+        public string FeedLanguageCode => "fr";
+
+        public bool Filter(SyndicationItem item) =>
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/JeromeGiacomini.cs]: Remove the filter. We already filter for Xamarin by default and you don't implement `IFilterMyBlogPosts`.

Comment (jgiacomini) [reply src/Firehose.Web/Authors/JeromeGiacomini.cs]: If you worried about my activity maybe I can reassure you.

The last year I had 2 blogs : 
- https://jeromegiacomini.net/Blog/ (my perso)
- https://blogs.infinitesquare.com/users/jgiacomini (my pro blog)
I wrote also a book on Xamarin : https://www.editions-eni.fr/supports-de-cours/livre/xamarin-developpez-vos-applications-multiplateformes-pour-ios-android-et-windows-9782409007477

This year each post I will wrote on my pro blog=> i will also put them on perso blog. :)

Comment (jgiacomini) [reply src/Firehose.Web/Authors/JeromeGiacomini.cs]: My book is now finished I will have more time to write article. :)

**Comment by Cheesebaron** [comment]
Welcome to PlanetXamarin.

Also an assignment for you: Add HSTS and rules to make `http` resources on your blog try upgrade to `https`.

This _could_ be as easy as adding: 

```html
  <meta http-equiv="Content-Security-Policy" content="default-src https:">
  <meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">
```

**Comment by jgiacomini** [comment]
Now when you wantt to use my blog in http => you are redirected to https url. :)

**Comment by jgiacomini** [comment]
Just to know : when the website will be deploy with me as author? :)
Thank by advance for your answer. :)

**Comment by Cheesebaron** [comment]
It should have done that already. Will look into the issue.

**Comment by Cheesebaron** [comment]
Opened #466

**Comment by jgiacomini** [comment]
Thanks cheesebaron. I will wait before to publish new articles to be on website.

**Pull request merged**